core/Core.php

```diff
@@ -113,7 +113,12 @@
 }
 
 // Now that the class manifest is up, load the static configuration
-$configManifest = new SS_ConfigStaticManifest(BASE_PATH, false, $flush);
+if (defined('SS_CONFIGSTATICMANIFEST')) {
```

Comment:
This needs documentation, both in the ss_enviroment setting list, and in the 3.4.0 changelog please. :)